dw-surface.js

```diff
@@ -171,6 +171,11 @@ export class DwSurface extends LitElement {
       elevation: {
         type: Number,
         reflect: true
+      },
+
+      noBoxShadow: {
```

Comment:
If a property isn't needed  (e.g. reactive rendering) for this change, shouldn't introduce a new property.